### ISSUE TITLE
Enable PFIO server in applications use GEOS.x and MAPL3G Cap

### ISSUE THREAD

**Issue by bena-nasa** [GEOS-ESM/MAPL#2740]
The new MAPL3G version of History will still presumably for the time being the PFIO server as the vehicle to output data at the end of the day. The first step then is figuring where and how to enable the PFIO server in applications using our generic GEOS.x driver and the MAPL3G Cap.F90.

**Comment by tclune** [comment]
Yes - this is important, and nontrivial.  Hopefully we can find some intermediate process that lets you make progress in History without requiring the server just yet.

**Comment by bena-nasa** [comment]
@tclune I think we will need this sooner rather than later, not like next week, but I think we want to be working on this actively. 

Definitely intermediate stuff, but since the coupling layer is doing most of the work, once we have the fields, either from a single, non-vector field, or an expression involving non-vector fields there's not a whole lot to do before it's time to write other than decide when to write, or maybe evaluate an expression. 

In other words I just don't see it as a long process to where we get the point where we say, there's nothing else to do with these fields the collection has, let's write something.

**Comment by tclune** [comment]
I understand.   If we can at least avoid the server issue initially it will help.

We need to review the design document (ppt) we have for I/O in general and start implementing enough of that to work.  Largely leveraging existing gridded_io and pfio, but I think we can get something even simpler (and definitely cleaner) than in MAPL2.
